### PR TITLE
Added a new calendar for Moja Global Community Website

### PR DESCRIPTION
**Pull request by Palaksharma23**
I have added a calendar for meetings section of moja-global Community website using react big calendar module

### CODE REVIEW

**Comment by Palaksharma23** [comment]
@HarshCasper @sohamsshah sir can you please once review it .

File: website/src/pages/initiatives.js [hunk 1]
```diff
@@ -2,9 +2,28 @@ import React from 'react';
...
+const localizer = dateFnsLocalizer({
+  format,
+  parse,
+  startOfWeek,
+  getDay,
+  locales,
+});
+// format for adding a meeting is new Date(year , month-1 , date)
+function ondouleclick() {
+  window.open('https://workspace.google.com/intl/en_in/products/calendar/?utm_source=google&utm_medium=cpc&utm_campaign=1011352-Workspace-APAC-IN-en-BKWS-EXA-Golden&utm_content=text-ad-none-none-DEV_c-CRE_469259700126-ADGP_Hybrid%20%7C%20AW%20SEM%20%7C%20BKWS%20~%20EXA%20%7C%20Calendar%20%7C%20%5BM:1%5D%20%7C%20IN%20%7C%20EN-KWID_43700057535047290-kwd-1844967558&userloc_1007820-network_g&utm_term=KW_google%20calendar&gclid=CjwKCAiAz--OBhBIEiwAG1rIOkbt8UI2XpXuSfpJzoHLn12sVQllOHJEgzju9YKwJOe1AgCPEgUKMBoCnK8QAvD_BwE&gclsrc=aw.ds', '_blank');
```
Comment (sohamsshah) [inline website/src/pages/initiatives.js:25]: This Link should take the user to the corresponding event in google calendar. Need to replace this google calendar **meeting** link for Moja Global. We can export this static data from the `websiteData/meetings.js` folder. 

Alternatively, if we can find some way to link a particular meeting from the google calendar, we can take the data from the events.obj from `websiteData/meetings.js`

File: website/src/pages/initiatives.js [hunk 2]
```diff
@@ -72,30 +91,26 @@ const Community = () => {
...
-                itemSize={100}
-                width="90%"
-                itemData={{ isCompleted: false }}
-              >
-                {MeetingCard}
-              </List>
+          <div>
+            <div>
+              <div className={styles.addToCalendar}>
+                <a href="https://workspace.google.com/intl/en_in/products/calendar/?utm_source=google&utm_medium=cpc&utm_campaign=1011352-Workspace-APAC-IN-en-BKWS-EXA-Golden&utm_content=text-ad-none-none-DEV_c-CRE_469259700126-ADGP_Hybrid%20%7C%20AW%20SEM%20%7C%20BKWS%20~%20EXA%20%7C%20Calendar%20%7C%20%5BM:1%5D%20%7C%20IN%20%7C%20EN-KWID_43700057535047290-kwd-1844967558&userloc_1007820-network_g&utm_term=KW_google%20calendar&gclid=CjwKCAiAz--OBhBIEiwAG1rIOkbt8UI2XpXuSfpJzoHLn12sVQllOHJEgzju9YKwJOe1AgCPEgUKMBoCnK8QAvD_BwE&gclsrc=aw.ds">
```
Comment (sohamsshah) [inline website/src/pages/initiatives.js:97]: Need to replace this google calendar link for Moja Global. We can export this static data from the `websiteData/meetings.js` folder.

File: website/src/pages/initiatives.js [hunk 3]
```diff
@@ -2,9 +2,28 @@ import React from 'react';
...
+};
+const localizer = dateFnsLocalizer({
+  format,
+  parse,
+  startOfWeek,
+  getDay,
+  locales,
+});
+// format for adding a meeting is new Date(year , month-1 , date)
+function ondouleclick() {
```
Comment (sohamsshah) [inline website/src/pages/initiatives.js:24]: Please rename this function as `onDoubleClick()`

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Great work @Palaksharma23 . I have suggested few changes, otherwise the PR looks good.

Before merging this, we can do a small research around embedding `Google Calendar` to the project (assuming the Google Calendar of Moja Global would be public, shareable and open for anyone who wants to join.) The embed is a typical `iframe` and also the theme and other actions can be updated.

If the embed works satisfactorily, we can consider this alternative over `react-big-calendar` to avoid an external dependency and overhead of updating meeting schedule manually on the website.

Refer: https://support.google.com/calendar/answer/41207

Cc: @HarshCasper

**Comment by sohamsshah** [comment]
Meetings Section now looks like:
image

We can add the meetings inside `websiteData/meetings.js`. Have added comments on how to add a meeting.

**Review by HarshCasper — APPROVED** [review]
Cool work @Palaksharma23 and @sohamsshah 🎉 

LGTM!

**Pull request merged**